mapstruct-2/src/test/java/com/baeldung/mapstruct/iterable/DepartmentMapperTest.java

```diff
@@ -0,0 +1,52 @@
+package com.baeldung.mapstruct.iterable;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.baeldung.mapstruct.iterable.model.Department;
+import com.baeldung.mapstruct.iterable.model.Employee;
+
+public class DepartmentMapperTest {
+
+    private final DepartmentMapper departmentMapper = DepartmentMapper.INSTANCE;
+    private final List<Employee> employees = Arrays.asList(
+        new Employee("John", "(555)1111-1234"),
+        new Employee("Jane", "(555)2222-4321")
+    );
+
+    @Test
+    public void givenListOfEmployees_whenMapDepartment_thenAssertMappingIsSuccessful() {
```

Comment:
If using Junit5, we can drop the public modifiers on test methods

Reply:
Agree, dropped.